### ISSUE TITLE
Skribisto set-up goes off-screen on GNOME desktop when using 200% scaling

### ISSUE THREAD

**Issue by chloe-everhart** [jacquetc/skribisto#213]
Hi.  I just tried Skribisto.  As a fellow Esperantist, I always love trying software named in Esperanto.  I'm "low vision" and use Pop! OS with 200% scaling on their GNOME desktop.  In the set-up screen, it seemed like it ran off the bottom of the screen, and I couldn't get to all of the options.  

I tried to uninstall and reinstall it to get a screenshot of that, but it conserved the choices I made so I couldn't get back to the first screen to get a screen shot for you.

**Comment by jacquetc** [comment]
Hello, 
Thank you for trying Skribisto in an uncommon way. I have an idea of what is happening and I keep that in mind.

However, since September, I am working on a more useful user interface for desktop computers. Less sparkles and more serious. If you want, I can give you a way test the beta software. And at the same time, I'd be glad if you can test it with your low vision parameters :-)

**Comment by chloe-everhart** [comment]
Testing a focused, distraction free user interface sounds nice.  

I recently discovered FreeTube, which I guess is built to encourage focus/eliminate distraction.  (being easily distracted and losing track of what I'm doing/really high transition time costs, is another problem I struggle with.)  I love apps that help me learn and focus, and make that less of a problem for me instead of exploiting my weakness to try to sell me stuff.  

Does the user interface have a very effective blue light filter built in, or will work with the Night LIght feature in GNOME?  My eyes are extremely sensitive to some kinds of blue and white light.  On Pop!, I have night light set to Always On, and the slider set about 3/4 of the way to "More Warm".   Concurrently, I also use the built in blue light filter in the eye-care monitor I have on the highest setting.  

I use an e-ink cell phone, the Hisense A5, and soon will be getting the Hisense A7cc.  My house is lit by upright halogen torchiere lamps. 

The University of Toronto did some interesting research on how blue light can permanently bleach melonopsin (sp? receptors.  It seems like it's pretty much an instant thing for me and I get "screen burn", and see the image for a long time after my eyes accidentally get exposed to the wrong type of light.  

In short, yes, I'm interested, and thank you for offering a way to be of service to FOSS.

**Comment by jacquetc** [comment]
Hello,

The "distraction free" part is not yet implemented, but it's coming. You will have the liberty to switch between a light and dark theme, or create your own theme if needed. I own a Dasung 13,3" E-Ink monitor when my eyes are tired. 

I don't know about "blue light filter" or the "Night LIght feature". My software heavily depends on the Qt framework, which,in turn, tries to play nice with the Deskop Environment (Gnome, KDE...).


Here is the test build for Skribisto:
In a terminal/console: 
`flatpak install --user https://dl.flathub.org/build-repo/1161/eu.skribisto.skribisto.flatpakref`

A few commands to help you:
to run the beta of Skribisto :
flatpak run --branch=test eu.skribisto.skribisto
to list the installations :
flatpak list
to run the older but stable Skribisto :
flatpak run --branch=stable eu.skribisto.skribisto
to remove it:
flatpak remove eu.skribisto.skribisto
and you select the "test" skribisto so as to remove it
And could you please report any bug ?
Thank you

**Comment by chloe-everhart** [comment]
These are the notes I made, so you could have my first experience, as I went through the process.  

Install: I usually install stuff through the store, so that was kinda cool to watch.  I felt a brief moment of friction when control + v didn’t work and I had to use copy paste and the mouse, but that’s an issue for another program.  Installed and started easily with the instructions  you gave.  

Upon opening, I noticed gray text in the middle of the page, but probably not immediately.  I had to lean in to see what the text said.  At first I tried typing and realized nothing happened, then leaned in to read it.  

There were some buttons there that I hovered over.  I have my computer bogged down right now, but despite leaving the mouse over those buttons for a few sections, it never told me what they did.  Apparently they changed the view somehow.  Took another screenshot.

So start by opening a document to work in...  Buttons are on the page.  I used to just expect that.  Now it’s a thing that brings sigh of relief.  

So many dialog boxes have them off the bottom of the screen, and then I have to decide if what I was trying to do is worth changing a bunch of settings twice and using the shaky zoom feature on the unscaled setting.  I prefer scaling because I get motion sick when I use screen zoom a lot, and it usually jumps when you start typing something.

Okay, so I started a project and that was easy.  I clicked the thing in the middle of the page at the top, and it brought me to a book outline.  Is this something I would want to use to write a letter or something?  I didn’t plan on having scenes in my project, which I though could be a plan stuffed into the template of a novel.  Can I delete that part, or does it mean something else to people who write novels?

These questions don’t need answers  They’re stuff I’ll figure out while I’m trying it, probably.  This was just what went through my mind as I opened and tried it for the first time, and maybe it’ll be useful as you design it. 

PS.  I also like that I could move the column to make my whole long title visible. 

Scr
![Screenshot from 2023-01-21 03-23-10](https://user-images.githubusercontent.com/68519373/213867867-4c1f680e-caa9-47fa-86ee-bb68af898ebf.png)
eenshots.
![Scre
![Screenshot from 2023-01-21 03-33-46](https://user-images.githubusercontent.com/68519373/213867949-d78fb2f5-1173-4cab-b565-e2bb87e7ece7.png)

![Screenshot from 2023-01-21 03-28-14](https://user-images.githubusercontent.com/68519373/213867947-4f2425ec-8025-47f1-a63a-8cd3e385f1a4.png)
enshot from 2023-01-21 03-39-00]

![Screenshot from 2023-01-21 03-39-00](https://user-images.githubusercontent.com/68519373/213868066-dfafa3fb-86ea-4a87-9b34-1b9c8b2987ae.png)

**Comment by jacquetc** [comment]
I am the first to admit that the experience for a newcomer is somewhat rough. I concentrated on reliability and core functionalities. I'll add "Start-up dialog" later. 

You are free to rename all the elements to fit your needs. Also, play with the theme creator in the preferences.

**Comment by chloe-everhart** [comment]
I started making a document with no theme to test making my own theme.  There's three dots.  I think it's a menu button, but when I hover on it and/or click on it, nothing happens.  I can't figure out how to add a book intro, chapters or scenes. I couldn't find an option in the top menu that would let me do it.  I'm stumped.  How do I add those?

-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



Jan 21, 2023, 06:19 by ***@***.***:
